### PR TITLE
feat: add load_balancing_anomaly_mitigation

### PR DESCRIPTION
**Pull request by mschfh**
## what

Add [load_balancing_anomaly_mitigation](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lb_target_group#load_balancing_anomaly_mitigation-1) variable

## references

[lb_target_group](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lb_target_group)
https://github.com/cloudposse/terraform-aws-alb/pull/193

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -40,7 +40,18 @@ variable "deregistration_delay" {
 variable "load_balancing_algorithm_type" {
   type        = string
   default     = "round_robin"
-  description = "Determines how the load balancer selects targets when routing requests. Only applicable for Application Load Balancer Target Groups. The value is round_robin or least_outstanding_requests. The default is round_robin."
+  description = "Determines how the load balancer selects targets when routing requests. Only applicable for Application Load Balancer Target Groups. The value is round_robin, least_outstanding_requests or weighted_random. The default is round_robin."
+}
+
+variable "load_balancing_anomaly_mitigation" {
+  type        = bool
+  default     = false
```
Comment (nitrocode) [inline variables.tf]: Shouldn't this be a string and default to null or off ?

Comment (mschfh) [reply variables.tf]: Yes, it should be. Thanks for spotting it, fixed it.

**Comment by gberenice** [comment]
@mschfh I know it's not related to your contribution, but to pass the tests, we must update the AWS account value, as it's done here https://github.com/cloudposse/terraform-aws-ecs-alb-service-task/pull/252/files
Could you please do that so I can approve your work? Much appreciate you!

**Comment by mschfh** [comment]
@gberenice done.

Thanks for the quick review!

**Review by gberenice — APPROVED** [review]

**Pull request merged**